packages/manager/modules/carbon-calculator/src/dashboard/routing.js

```diff
@@ -8,6 +10,12 @@ export default /* @ngInject */ ($stateProvider) => {
       dashboardLink: /* @ngInject */ ($state) => $state.href('app.dashboard'),
       breadcrumb: /* @ngInject */ ($translate) =>
         $translate.instant('carbon_calculator_my_carbon_footprint'),
+      trackClick: /* @ngInject */ (atInternet) => {
+        atInternet.trackClick({
+          name: TRACKING_NAME,
+          type: 'action',
+        });
+      },
```

Comment:
**nitpick:** If you want to track page, `trackClick` is not the right choice. I think you should use `trackPage` or define `atInternet: { rename: '...'}` on your state definition.

Reply:
Second option is better, I will do it